rasa/core/policies/ted_policy.py

```diff
@@ -112,13 +117,17 @@
 from rasa.shared.core.events import EntitiesAdded, Event
 from rasa.shared.nlu.training_data.message import Message
 from rasa.shared.utils import io as shared_io_utils
+from rasa.core.policies._ted_policy import TEDPolicy
 
-if TYPE_CHECKING:
-    from rasa.shared.nlu.training_data.features import Features
+from rasa.shared.nlu.training_data.features import Features
 
 
 logger = logging.getLogger(__name__)
 
+# This is a workaround around until we have all components migrated to `GraphComponent`.
+# (see the top of the file for more information).
```

Comment:
There is no information at top of this file.

Reply:
🤦🏻